### ISSUE TITLE
Error: No named parameter with the name 'selectedItem'.

### ISSUE THREAD

**Issue by robertoltrocha** [codegrue/card_settings#169]
Hi.
I'm updating my third project and I'm not able to compile on android. Any suggestions for the error below? Thank you for your help.
I have already applied:

> flutter clean
> invalidate and restart
> delete pubspec.lock file

```
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/card_settings-2.0.1/lib/widgets/picker_fields/card_settings_list_picker.dart:165:7: Error: No named parameter with the name 'selectedItem'.
      selectedItem: option,
      ^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/flutter_material_pickers-2.1.1/lib/helpers/show_scroll_picker.dart:8:6: Context: Found this candidate, but the arguments don't match.
void showMaterialScrollPicker({
     ^^^^^^^^^^^^^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/card_settings-2.0.1/lib/widgets/picker_fields/card_settings_radio_picker.dart:166:7: Error: No named parameter with the name 'selectedItem'.
      selectedItem: option,
      ^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/flutter_material_pickers-2.1.1/lib/helpers/show_radio_picker.dart:8:6: Context: Found this candidate, but the arguments don't match.
void showMaterialRadioPicker({
     ^^^^^^^^^^^^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/card_settings-2.0.1/lib/widgets/picker_fields/card_settings_selection_picker.dart:173:7: Error: No named parameter with the name 'selectedItem'.
      selectedItem: option,
      ^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/flutter_material_pickers-2.1.1/lib/helpers/show_selection_picker.dart:8:6: Context: Found this candidate, but the arguments don't match.
void showMaterialSelectionPicker({
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/card_settings-2.0.1/lib/widgets/picker_fields/card_settings_checkbox_picker.dart:115:7: Error: No named parameter with the name 'selectedItems'.
      selectedItems: value,
      ^^^^^^^^^^^^^
../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/flutter_material_pickers-2.1.1/lib/helpers/show_checkbox_picker.dart:8:6: Context: Found this candidate, but the arguments don't match.
void showMaterialCheckboxPicker({
     ^^^^^^^^^^^^^^^^^^^^^^^^^^


FAILURE: Build failed with an exception.

* Where:
Script 'C:\src\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1035

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\src\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 18s
Exception: Gradle task assembleDebug failed with exit code 1
```

```
[√] Flutter (Channel beta, 2.2.0-10.1.pre, on Microsoft Windows [versÃ£o 10.0.19042.928], locale pt-BR)
    • Flutter version 2.2.0-10.1.pre at C:\src\flutter
    • Framework revision 0941968447 (23 hours ago), 2021-04-15 12:01:02 -0700
    • Engine revision d2a2e93510
    • Dart version 2.13.0 (build 2.13.0-211.6.beta)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\rocha\AppData\Local\Android\sdk
    • Platform android-S, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio1\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio1
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code, 64-bit edition (version 1.53.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.20.0

[√] Connected device (1 available)
```

**Comment by robertoltrocha** [comment]
I tried with flutter stable version but no success

```
[√] Flutter (Channel stable, 2.0.5, on Microsoft Windows [versÃ£o 10.0.19042.928], locale pt-BR)
    • Flutter version 2.0.5 at C:\src\flutter
    • Framework revision adc687823a (2 hours ago), 2021-04-16 09:40:20 -0700
    • Engine revision b09f014e96
    • Dart version 2.12.3

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\rocha\AppData\Local\Android\sdk
    • Platform android-S, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio1\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio1
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code, 64-bit edition (version 1.53.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.20.0

[√] Connected device (1 available)
    • SM G975F (mobile) •  android-arm64 • Android 11 (API 30)

• No issues found!
```

**Comment by codegrue** [comment]
Working on a fix now.

**Comment by codegrue** [comment]
Fixed in 2.0.2
